packages/firestore/test/util/api_helpers.ts

```diff
@@ -57,6 +57,17 @@ export function firestore(): Firestore {
   return FIRESTORE;
 }
 
+export function namedFirestore(projectId: string): Firestore {
+  return new Firestore(
```

Comment:
You don't need the name, do you? I think you just need to return a new instance. This could just be `newTestFirstore()`.

Reply:
done.